modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDataStore.java

```diff
@@ -117,6 +117,8 @@
 
     long maxDbfSize = ShapefileFeatureWriter.DEFAULT_MAX_DBF_SIZE;
 
+    private boolean tryCpgFile = false;
```

Comment:
Common Java naming approach is to keep acronyms in all caps. See also: https://stackoverflow.com/questions/2236807/java-naming-convention-with-acronyms#:~:text=From%20sun%20java%20docs%3A,such%20as%20URL%20or%20HTML).

```suggestion
    private boolean tryCPGFile = false;
```